ocs_ci/framework/__init__.py

```diff
@@ -303,7 +303,7 @@ def get_consumer_indexes_list(self):
         """
         consumer_indexes_list = []
         for i, cluster in enumerate(self.clusters):
-            if cluster.ENV_DATA["cluster_type"] in ["consumer", "hci_client"]:
+            if cluster.ENV_DATA["cluster_type"] in ["consumer", "hci_client", "Client"]:
```

Comment:
is the `Client` really with Capital `C` ? And where it comes from?  Do we have some reference?
I didn't find cluster_type set to `Client` in the code.

After offline discussion it's passed from job config. We can update there, but for now I suggest to change to:
```suggestion
            if cluster.ENV_DATA["cluster_type"].lower() in ["consumer", "hci_client", "client"]:
```

If cluster.ENV_DATA["cluster_type"] can be something else than string, we need to update wiht:
`cluster.ENV_DATA.get("cluster_type", "")`

Reply:
resolved